legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/domain/api/review/ReviewApi.java

```diff
@@ -46,12 +47,20 @@
      * @param projectId   project id
      * @param state       review state
      * @param revisionIds a set of revision IDs, with each we will get the reviews are associated
+     * @param workspaceId workspace Id with which review is associated
+     * @param workspaceType workspace type with which review is associated
      * @param since       this time limit is interpreted based on the chosen state, for example: if only committed reviews are fetched, 'since' will concern the committed time
      * @param until       this time limit is interpreted based on the chosen state, for example: if only committed reviews are fetched, 'since' will concern the committed time
      * @param limit       maximum number of reviews to get
      * @return reviews
      */
-    List<Review> getReviews(String projectId, ReviewState state, Iterable<String> revisionIds, Instant since, Instant until, Integer limit);
+    List<Review> getReviews(String projectId, ReviewState state, Iterable<String> revisionIds, WorkspaceType workspaceType, Predicate<String> workspaceId, Instant since, Instant until, Integer limit);
```

Comment:
```suggestion
    List<Review> getReviews(String projectId, ReviewState state, Iterable<String> revisionIds, Predicate<String> workspaceId, Set<WorkspaceType> workspaceTypes, Instant since, Instant until, Integer limit);
```